<launch>
  <group ns="plug_detector" clear_params="true">
    <!-- ROI settings -->
    <!--param name="roi_policy" type="str" value="LastImageLocation"/-->
    <param name="roi_policy" type="str" value="FullResolution"/>
    <!--
    <param name="roi_policy" type="str" value="TargetFrame"/>
    <param name="target_frame_id" type="str" value="r_gripper_tool_frame"/>
    -->
    <param name="resize_factor_found" type="double" value="3.0"/>
    <param name="resize_factor_failed" type="double" value="1.2"/>
    <param name="target_roi_size" type="int" value="800"/>

    <!-- Plug checkerboard -->
    
    <param name="square_size" type="double" value="0.0042"/>
    <param name="board_width" type="int" value="4"/>
    <param name="board_height" type="int" value="5"/>
    
    <!-- Calibration checkerboard -->
    <!--
    <param name="square_size" type="double" value="0.0225"/>
    <param name="board_width" type="int" value="6"/>
    <param name="board_height" type="int" value="9"/>
    -->
  </group>
  <node pkg="outlet_detection" type="plug_node" respawn="false" output="screen"/>
</launch>
